android/assets/jsons/translations/Dutch.properties

```diff
@@ -643,19 +639,19 @@ Base happiness = Geluksbasis
 Occupied City = Bezette Stad
 Buildings = Gebouwen
 
-# For the "when constructing [military units]" translation
+when constructing [military units] = als er [military units] gebouwd worden
 Military = militair
 military units = militaire eenheden
 melee units = slageenheden
 mounted units = bereden eenheden
 naval units = marine eenheden
 ranged units = afstandseenheden
-# For the All "newly-trained [relevant] units in this city receive the [] promotion" translation. Relevant as in 'units that can receive'
+newly-trained [relevant] units in this city receive the [] promotion = nieuw getrainde [military units] in deze stad ontvangen de [] promotie
 relevant = relevante
 non-air = niet-lucht
-# For '[stats] from [Water] tiles in this city'
+[stats] from [Water] tiles in this city = [stats] van [Water] tegels in deze stad
 Water = Water
-# For [stats] from [Water resource] tiles in this city
+[stats] from [Water resource] tiles in this city = [stats] van [Water resource] tegels in deze stad
```

Comment:
Alle regels die met een # beginnen zijn bedoeld als commentaar en hoeven in principe niet vertaald te worden, het geeft voornamelijk uitleg over wat er met de vertalingen eronder gebeurd